core/src/main/resources/lib/form/select/select.js

```diff
@@ -5,9 +5,12 @@ function updateListBox(listBox,url,config) {
     config = object(config);
     var originalOnSuccess = config.onSuccess;
     var l = $(listBox);
-    var status = findFollowingTR(listBox, "validation-error-area").firstChild.nextSibling;
-    if (status.firstChild && status.firstChild.getAttribute('data-select-ajax-error')) {
-        status.innerHTML = "";
+    var followingTR = findFollowingTR(listBox, "validation-error-area");
+    if(followingTR!=undefined){
+    	var status = findFollowingTR(listBox, "validation-error-area").firstChild.nextSibling;
```

Comment:
Use the existing field instead of repeating `findFollowingTR ()` ?

Reply:
This has been taken care of by having an exception handler in place Oleg. Ideally, it reduces the overhead of using findFollowingTR () twice.